src/extractNumber.ts

```diff
@@ -0,0 +1,16 @@
+import type { Transformer } from 'integreat'
+import { isNumber, isString } from './utils/is.js'
+
+const extractAndParseInt = (value: string) => {
+  const digits = value.replace(/\D/g, '')
+  return digits ? parseInt(digits, 10) : undefined
+}
+
+const transformer: Transformer = () => () => (value) =>
+  isString(value)
+    ? extractAndParseInt(value)
+    : isNumber(value)
```

Comment:
What did you think about this, @GoodGuyGoose? :)

Reply:
The end result will be exactly the same for now since only digits can be extracted from the string.  From the consistency perspective I think the difference is negligible.  However, I think there is value in using `parseNum()` if we decide to add functionality such as flags, that can cause the extraction of floats, e.g. `extractFloat: true`, in the future.  In this case it's valuable to already be using `parseNum()` so we are already on the correct track. :)

Reply:
That was my reasoning behind actually implementing the update you proposed. @kjellmorten

Reply:
From conversation outside the PR: There is another number parser function in this package that also coerces other type of values to numbers. I though we were talking about that one, but this is fine as it is now.